app/src/main/java/app/nexd/android/ui/MainActivity.kt

```diff
@@ -41,6 +41,13 @@ class MainActivity : AppCompatActivity() {
         api = Api()
         api.setBearerToken(Preferences.getToken(this))
         hideKeyboardOnTouch()
+
+        val input = EditText(applicationContext)
+        val lp = LinearLayout.LayoutParams(
+            LinearLayout.LayoutParams.MATCH_PARENT,
+            LinearLayout.LayoutParams.MATCH_PARENT
+        )
+        input.layoutParams = lp
```

Comment:
```suggestion
        val input = EditText(applicationContext)
        val lp = LinearLayout.LayoutParams(
            LinearLayout.LayoutParams.MATCH_PARENT,
            LinearLayout.LayoutParams.MATCH_PARENT
        )
        input.layoutParams = lp
```

why do we need this?